src/frontend/package.json

```diff
@@ -16,7 +16,10 @@
     "lint-all": "yarn lint-fix && yarn prettier-write",
     "build-sass": "sass scss/_main.scss ../backend/marsha/static/css/main.css --load-path=node_modules",
     "build-sass-production": "sass scss/_main.scss ../backend/marsha/static/css/main.css --style=compressed --load-path=node_modules",
-    "test": "jest"
+    "test": "jest",
+    "test:generate-output": "jest --json --outputFile=.jest-test-results.json",
```

Comment:
this file should be committed or ignored ?

Reply:
Not sure yet, this is used for displaying tests results in components page inside storybook.
Personally, I don't find it very useful, but I may have missed something.
I'll ignore it for now.

Reply:
What the purpose of this command ? Is it mandatory to have it ?

Reply:
It comes from this addon https://www.npmjs.com/package/@storybook/addon-jest
I've installed it for experimentation purpose, not sure how to effectively use it for now.
Maybe we should remove it ?

Reply:
I'll remove it as it is unused for now.
We should install addons as our needs evolves around storybook.